src/components/ErrorPage.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+    <section class="page_404">
+      <div class="container">
+        <div class="row">
+          <div class="col-sm-12">
+            <div class="col-sm-10 col-sm-offset-1 text-center">
+              <div class="four_zero_four_bg">
+                <h1 class="text-center">404</h1>
+              </div>
+  
+              <div class="contant_box_404">
+                <h3 class="h2">
```

Comment:
```suggestion
                <h2 class="h2">
```
Skipping heading levels (h1 > h3) is not good for accessibility: the screen reader users use headings and heading levels to navigate the page.